rightscript.go

```diff
@@ -835,6 +835,46 @@ func validateRightScript(file string, ignoreMissingMetadata bool) (*RightScript,
 	return &rightScript, nil
 }
 
+func rightScriptCommit(href, message string) {
+	var (
+		rightScriptName string
+		err             error
+	)
+
+	rightScriptName, err = getRightScriptNameFromInput(href)
+	if err != nil {
+		fatalError("%s", err.Error())
+	}
+
+	foundID, err := rightScriptIdByName(rightScriptName)
+	if err != nil {
+		fatalError("Could not find rightscript ID with name: %s and href: %s", rightScriptName, href)
+	}
+	if foundID == "" {
+		fatalError("Could not find rightscript ID with name: %s and href: %s", rightScriptName, href)
+	} else {
+		commitRightScript(foundID, message)
+	}
+}
+
+func commitRightScript(foundID, message string) error {
+	client, _ := Config.Account.Client15()
+	var rightscriptLocator *cm15.RightScriptLocator
+
+	// found rightscript, do a commit
+	href := fmt.Sprintf("/api/right_scripts/%s", foundID)
+
+	params := cm15.RightScriptParam{
+		CommitMessage: message,
+	}
+	rightscriptLocator = client.RightScriptLocator(href)
+	err := rightscriptLocator.Commit(&params)
+	if err != nil {
+		return err
+	}
+	return err
```

Comment:
We're not really doing anything different here with the different branches, maybe it would be fine to just do:
```golang
return rightscriptLocator.Commit(&params)
```